client/assets/style.scss

```diff
@@ -1,13 +1,44 @@
-@import "tools/index";
-@import "base/index";
-@import "pages/index";
-@import "./toastr.scss";
+@import 'tools/index';
+@import 'base/index';
+@import 'pages/index';
+@import './toastr.scss';
 
-@import url("fonts.scss");
+@import url('fonts.scss');
 
 // google font
 @import url('https://fonts.googleapis.com/css?family=Spectral+SC');
 @import url('https://fonts.googleapis.com/css?family=Ubuntu');
+// Import Abel
```

Comment:
### Nitpick

Can you please remove the fonts above? We're not making use of it.

We are sticking to just `Abel`